frontend/src/metabase/public/lib/types.ts

```diff
@@ -36,6 +36,14 @@ export type EmbeddingDisplayOptions = {
   downloads: boolean | null;
 };
 
+export type AdditionalEmbeddingHashOptions = {
```

Comment:
Can you help me understand what is "additional" in this context and why these two are not in `EmbeddingDisplayOptions`?

Reply:
- embedding display options are the ones we have in static embed modal display options.
    ![image](https://github.com/user-attachments/assets/c08fb43b-5412-4bce-8cdd-c830c767baef)
- dashboards have their own hash options that don't apply to questions `refresh` and `fullscreen` that are used to control the dashboard state.
     https://github.com/metabase/metabase/blob/e840233a4f950d0180333c213fa8b6478b391852/frontend/src/metabase/dashboard/types/hash-options.ts#L3-L11
- Both questions and dashboards have these common hash options that's why I called them `AdditionalEmbeddingHashOptions`

Reply:
Ah thanks, can we add some small comments about this in the code?

Reply:
Sure thing.

Reply:
Added in https://github.com/metabase/metabase/pull/49441/commits/5dd8fd3067d53431fd44b1050872226ba6ba5b4a